napari/components/_interaction_box_mouse_bindings.py

```diff
@@ -258,3 +415,16 @@ def _on_end_newbox(self, viewer, event):
             self._interaction_box_model.selection_box_final = (
                 self._interaction_box_model._box[Box.WITHOUT_HANDLE]
             )
+
+    def _reset_active_layer_affine(self, event=None):
+        active_layer = self._viewer.layers.selection.active
+        if active_layer is not None:
+            active_layer.affine = Affine(ndim=active_layer.ndim)
+
+    def _transform_active_layer(self, event=None):
+        active_layer = self._viewer.layers.selection.active
+        if active_layer is not None and hasattr(active_layer, 'mode'):
+            if active_layer.mode != 'transform':
+                active_layer.mode = 'transform'

```

Comment:
If the active layer is a `Labels` layer and I press ctrl+shift+a, I get an error here because `'transform'` is not a valid mode for the labels layer. Would it be better to use something like `isinstance(active_layer, Image)` here instead?

Reply:
I instead went with enabling the `transform` mode in all layers that have a concept of mode. Happy to revert that ang do with the approach you described since it touches code in a lot of places. But it seems quite innocent to me and as a result enables the interaction mode in the shapes, points, and label layer.

Reply:
Sounds good to me!